src/lib/containers/TotalQueryResults.tsx

```diff
@@ -55,300 +18,19 @@ export type TotalQueryResultsProps = {
-      ])
-    }
-  }, [data, transformEnumFacetsForSelectionDisplay])
-
-  const removeFacetSelection = (facetWithSelection: FacetWithSelection) => {
-    const { facet, selectedValue } = facetWithSelection
-    if (facet.facetType === 'enumeration') {
-      applyChangesToValuesColumn(
-        getLastQueryRequest(),
-        facet,
-        applyChanges,
-        selectedValue?.value,
-        false,
-      )
-    } else {
-      applyChangesToRangeColumn(getLastQueryRequest(), facet, applyChanges, [
-        RadioValuesEnum.ANY,
-        RadioValuesEnum.ANY,
-      ])
-    }
-  }
-
-  const removeColumnSearchQuerySelection = (
-    columnName: string,
-    value: string,
-  ) => {
-    const cloneLastQueryRequest = cloneDeep(getLastQueryRequest())
-    if (!cloneLastQueryRequest.query.additionalFilters) {
-      return
-    }
-    cloneLastQueryRequest.query.additionalFilters =
-      cloneLastQueryRequest.query.additionalFilters?.reduce(function (
-        filtered: QueryFilter[],
-        el: QueryFilter,
-      ) {
-        if (
-          isColumnSingleValueQueryFilter(el) ||
-          isColumnMultiValueFunctionQueryFilter(el)
-        ) {
-          // For column-specific QueryFilters, filter out matching values on the matching column
-          const newElement = {
-            columnName: el.columnName,
-            values:
-              el.columnName === columnName
-                ? el.values.filter(el => el !== value)
-                : el.values,
-            operator: isColumnSingleValueQueryFilter(el)
-              ? el.operator
-              : undefined,
-            function: isColumnMultiValueFunctionQueryFilter(el)
-              ? el.function
-              : undefined,
-            concreteType: el.concreteType,
-          }
-
-          // Drop this QueryFilter if it has no more values
-          if (newElement.values.length > 0) {
-            filtered.push(newElement)
-          }
-        } else {
-          // Don't alter/drop non-column-specific QueryFilters
-          filtered.push(el)
-        }
-        return filtered
-      },
-      [])
-
-    executeQueryRequest(cloneLastQueryRequest)
-  }
-
-  const removeTextMatchesQueryFilter = (
-    queryFilter: TextMatchesQueryFilter,
-  ) => {
-    const cloneLastQueryRequest = cloneDeep(getLastQueryRequest())
-    if (!cloneLastQueryRequest.query.additionalFilters) {
-      return
-    }
-    cloneLastQueryRequest.query.additionalFilters =
-      cloneLastQueryRequest.query.additionalFilters.filter(
-        el =>
-          !(
-            isTextMatchesQueryFilter(el) &&
-            el.searchExpression === queryFilter.searchExpression
-          ),
-      )
-    executeQueryRequest(cloneLastQueryRequest)
-  }
-
-  const clearAll = () => {
-    const initQueryRequest = cloneDeep(getInitQueryRequest())
-    initQueryRequest.query.additionalFilters = []
-    executeQueryRequest(initQueryRequest)
-  }
-
-  const searchSelectionCriteriaPillProps:
```

Comment:
Transforming facets into pill props was cleaned up and moved to SelectionCriteriaPills.tsx